README.md

```diff
@@ -7,16 +7,17 @@
 ![](src/main/resources/META-INF/pluginIcon.svg)
 
 Git Machete plugin is a robust tool that simplifies your git related workflow.
-It's a great complement of IntelliJ version control system.
-You can with ease rebase, pull, push or reset branches.
-Just use discover action or manually define what relation between your branches you want to keep and leave the rest to Git Machete.
-It gives you a simple view of what's going on in your repository.
-Just look at the edge colors or status descriptions to see what should be rebased, pushed, pulled or reset.
+It's a great complement to the JetBrains products' built-in version control system.
+The "bird's eye view" provided by Git Machete makes merges/rebases/push/pulls hassle-free
+even when multiple branches are present in the repository
+(master/develop, your topic branches, teammate's branches checked out for review, etc.).
+A look at a Git Machete tab gives an instant answer to the question
+"what are the branches in this repository, what is going to be merged (rebased/pushed/pulled) and to what?".
```

Comment:
(yeah, I'm fixing myself here, sorry :trollface:)
```suggestion
"what branches are in this repository, what is going to be merged (rebased/pushed/pulled) and to what?".
```

Reply:
Generally, technical texts are *n o t o r i o u s* for multiple rounds of review, esp. as compared to code review. There're almost no automated checks (no compiler, no tests etc.) that could be applied to the text, as opposed to the code - so basically the human reviewers take over the role of the compiler :robot:

Reply:
See e.g. this one https://github.com/PawelLipski/docker-ci-blog/commits/master... 12 rounds with just one of the reviewers :exploding_head: